src/net/sourceforge/kolmafia/request/MallPurchaseRequest.java

```diff
@@ -157,6 +177,10 @@ public long getAvailableMeat() {
 
   @Override
   public String color() {
+    if (getForbiddenStores().contains(String.valueOf(this.shopId))) {
+      return "#c2002d";
```

Comment:
Can you get this colour from look and feel somehow to support those using dark modes